src/Accounts/Accounts.Test/AutosaveTests.cs

```diff
@@ -22,23 +22,38 @@
 using Xunit.Abstractions;
 using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
 using System;
+using System.Security;
 using Microsoft.Azure.Commands.Profile.Context;
 using Microsoft.Azure.Commands.ScenarioTest;
 using Microsoft.Azure.Commands.ResourceManager.Common;
 using Microsoft.Azure.Commands.TestFx.Mocks;
+using Moq;
 
 namespace Microsoft.Azure.Commands.Profile.Test
 {
     public class AutosaveTests
     {
         private MemoryDataStore dataStore;
         private MockCommandRuntime commandRuntimeMock;
+        private AzKeyStore keyStore;
         public AutosaveTests(ITestOutputHelper output)
         {
             XunitTracingInterceptor.AddToContext(new XunitTracingInterceptor(output));
             commandRuntimeMock = new MockCommandRuntime();
             dataStore = new MemoryDataStore();
-            ResetState();
```

Comment:
ResetState() never invokes, is it expected

Reply:
It is called at the beginning and end of the every test cases